### PR TITLE
v1.9.0

### PR DESCRIPTION
**Pull request by ReimarBauer**
Checklist
* [X] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [-] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ReimarBauer** [comment]
@ocefpaf seems I do some fundamental mistake, have you a hint for me?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -27,55 +30,60 @@ requirements:
     - future
     - menuinst  # [win]
   run:
-    - python
+    - python >=3.6
```
Comment (chrisburr) [inline recipe/meta.yaml]: This way of constraining the Python version should only be used for `noarch: python` packages. The version constraint here comes from using `skip: true  # [py27]`
```suggestion
    - python
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -27,55 +30,60 @@ requirements:
...
     - flask
     - flask-httpauth
     - werkzeug
+    - flask-socketio
+    - flask-sqlalchemy
+    - passlib
+    - gitpython
+    - psycopg2
+    - validate_email
+    - multidict
```
Comment (chrisburr) [inline recipe/meta.yaml]: From https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=84745&view=logs:

```
export PREFIX=/home/conda/feedstock_root/build_artifacts/mss_1572258270941/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl
export SRC_DIR=/home/conda/feedstock_root/build_artifacts/mss_1572258270941/test_tmp
import: 'mslib'
import: 'mslib'
+ mswms -h
Traceback (most recent call last):
  File "/home/conda/feedstock_root/build_artifacts/mss_1572258270941/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/bin/mswms", line 7, in <module>
    from mslib.mswms.mswms import main
  File "/home/conda/feedstock_root/build_artifacts/mss_1572258270941/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.6/site-packages/mslib/mswms/mswms.py", line 33, in <module>
    from mslib.mswms.wms import mss_wms_settings
  File "/home/conda/feedstock_root/build_artifacts/mss_1572258270941/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.6/site-packages/mslib/mswms/wms.py", line 57, in <module>
    from mslib.mswms.utils import conditional_decorator
  File "/home/conda/feedstock_root/build_artifacts/mss_1572258270941/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.6/site-packages/mslib/mswms/utils.py", line 30, in <module>
    import pint
ModuleNotFoundError: No module named 'pint'
```
I think you need:
```suggestion
    - multidict
    - pint
```

File: recipe/meta.yaml (hunk 1 above)
Comment (ReimarBauer) [reply recipe/meta.yaml]: ok

File: recipe/meta.yaml (hunk 2 above)
Comment (ReimarBauer) [reply recipe/meta.yaml]: thx, lets try. I missed that change

File: recipe/meta.yaml [hunk 3]
```diff
@@ -33,49 +36,55 @@ requirements:
...
+    - pint
 
 test:
```
Comment (chrisburr) [inline recipe/meta.yaml]: I think the `gitpython` package doesn't depend on `git` to allow people to have external installations. You can add it as a test dependency to fix the current failure:
```suggestion
test:
  requires:
    - git
```

Comment (ReimarBauer) [reply recipe/meta.yaml]: added git to the common requirements. I will discuss that in the gitpython-feedstock later.

I revoked the version tag, we fix first the main() problem but it looks now on a good path

**Comment by ocefpaf** [comment]
> @ocefpaf seems I do some fundamental mistake, have you a hint for me?

Thanks @chrisburr for the review. Sorry I missed your ping @ReimarBauer, I was away in a conference and still catching up with notifications.

**Pull request merged**